addons/website/views/snippets/s_references_social.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_social" name="References Social">
+    <section class="s_references_social o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div>
```

Comment:
```suggestion
                <div class="col-lg-12">
```
Looks like it takes full width by default, but I think it's cleaner to declare it 👍